live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -175,3 +175,19 @@
         default_user:
           name: {{ lookup('env', 'APPLIANCE_USERNAME') }}
           lock_passwd: false
+
+#
+# IPv6 addresses are not supported by the product thus we need
+# to ensure that they cannot be added manually by default. In
+# addition, we wouldn't want to have our secondary addresses
+# deleted when someone deletes the primary IP address of a
+# network interface, thus we enable the promotion of secondary
+# addresses to primary ones.
+#
+- lineinfile:
+    dest: /etc/sysctl.conf
```

Comment:
I think the common way to "extend" sysctl configuration is to add a file to /etc/sysctl.d/. There's a README in there explaining the naming convention of those files. Did you consider doing that instead of writing directly to /etc/sysctl.conf?